docs/pages/fragments/config-hooks.mdx

```diff
@@ -31,4 +31,5 @@ hooks:                              # struct[]  | Array of hooks to be executed.
     labelSelector: ...              # struct    | Key Value map of labels and values to select pods with
     containerName: ""               # string    | Container name to use after selecting a pod
     namespace: ""                   # string    | Kubernetes namespace to select pods in
+    once: ""                        # bool      | If true DevSpace will only execute the command once per deployment (default false)
```

Comment:
What does per deployment mean? Everytime the pipeline runs though? Maybe: `once - unless the targeted container is restarted which would allow re-executing this hook`? Not easy to express this accurately tbh

Reply:
Re-worded to `If true the command will execute once - until the container is re-deployed using DevSpace (default false)`.